compiler/src/main/scala/org/scalajs/nscplugin/GenJSExports.scala

```diff
@@ -786,7 +737,7 @@ trait GenJSExports[G <: Global with Singleton] extends SubComponent {
           else
             genApplyJSClassMethod(trgTree, defaultGetter, defaultGetterArgs)
         } else if (defaultGetter.owner == trgSym) {
-          /* We get here if a non-native constructor has a native compation.
+          /* We get here if a non-native constructor has a native companion.
```

Comment:
This change is in the wrong commit. It should be in the first commit, I believe.